src/Components/People.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+ class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchPeople: "",
+      currentPeople: {},
+      isError: false,
+      savedName: "",
```

Comment:
`this.state.savedName` is redundant since the same information lives inside of `this.state.currentPeople.name`. Try to re-write this component so that it doesn't use `this.state.savedName`!

This is about the 3rd step of thinking in react: trying to find a _minimal_ representation of UI state.